requirements/test_redirects.txt

```diff
@@ -0,0 +1,9 @@
+-r dev.txt
```

Comment:
OK. I realized that this is not installed in docker and only used in travis, correct?